lib/util/stream_utils.js

```diff
@@ -1445,3 +1469,44 @@ shaka.util.StreamUtils.DecodingAttributes = {
   POWER: 'powerEfficient',
   BANDWIDTH: 'bandwidth',
 };
+
+/**
+ * @private {!Set.<string>}
+ */
+shaka.util.StreamUtils.supportedImageMimeTypes = new Set()
+    .add('image/svg+xml')
+    .add('image/png')
+    .add('image/jpeg');
+
+/**
+ * @private {!Set.<string>}
+ */
+shaka.util.StreamUtils.unsupportedImageMimeTypes = new Set();
+
+/**
+ * @const {string}
+ * @private
+ */
+shaka.util.StreamUtils.minWebPImage = 'data:image/webp;base64,UklGRjoAAABXRU' +
+    'JQVlA4IC4AAACyAgCdASoCAAIALmk0mk0iIiIiIgBoSygABc6WWgAA/veff/0PP8bA//LwY' +
+    'AAA';
+
+/**
+ * @const {string}
+ * @private
+ */
+shaka.util.StreamUtils.minAvifImage = 'data:image/avif;base64,AAAAIGZ0eXBhdm' +
```

Comment:
I can't believe how large this tiny AVIF image is.  I played around in https://squoosh.app/ and only managed to shave it down from 311 bytes to 305.  :-(